### ISSUE TITLE
Integration with hyper don't work.

### ISSUE THREAD

**Issue by slyshykO** [kardeiz/jsonrpc-v2#5]
Try to build example/hyper.rs and receive following errors:

```
error[E0433]: failed to resolve: could not find `Chunk` in `hyper`                                                                                                                                  
    --> C:\Users\alex3\.cargo\registry\src\github.com-1ecc6299db9ec823\jsonrpc-v2-0.5.1\src\lib.rs:1182:34
     |
1182 |             let mut buf = hyper::Chunk::default();
     |                                  ^^^^^ could not find `Chunk` in `hyper`

error[E0599]: no method named `into_hyper_web_service` found for type `Server<R>` in the current scope                                                                                              
    --> C:\Users\alex3\.cargo\registry\src\github.com-1ecc6299db9ec823\jsonrpc-v2-0.5.1\src\lib.rs:1154:14
     |
679  | pub struct Server<R> {
     | -------------------- method `into_hyper_web_service` not found for this
...
1101 |     pub fn into_hyper_web_service(self: Arc<Self>) -> Hyper<R> {
     |            ---------------------- the method is available for `std::sync::Arc<Server<R>>` here
...
1154 |         self.into_hyper_web_service()
     |              ^^^^^^^^^^^^^^^^^^^^^^ help: there is a method with a similar name: `into_web_service`

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0433, E0599.
For more information about an error, try `rustc --explain E0433`.
error: could not compile `jsonrpc-v2`.

```

**Comment by kardeiz** [comment]
@slyshykO Thanks for the issue and the error log! I've fixed these two bugs in [`0.5.2`](https://crates.io/crates/jsonrpc-v2/0.5.2).

**Comment by slyshykO** [comment]
@kardeiz Thank you for your project.
The speed of fixing it is something unbelievable. 👍
